hwilib/devices/bitbox02.py

```diff
@@ -673,12 +679,20 @@ def script_config_from_utxo(
                 utxo = psbt_in.witness_utxo
             if utxo is None:
                 raise BadArgumentError("No utxo found for input {}".format(input_index))
-            if prevtx is None:
-                raise BadArgumentError(
-                    "Previous transaction missing for input {}".format(input_index)
-                )
```

Comment:
Shouldn't this check be retained for non-taproot scripts?

Reply:
We decided to remove it, since it is already done on the device